static/index.html

```diff
@@ -35,6 +35,15 @@
                     <a href="https://github.com/mattgodbolt/compiler-explorer">
                         <img height="20" width="20" src="assets/GitHub-Mark-20px.png">&nbsp;Source on GitHub</a>
                 </li>
+                <li class="dropdown">
+                    <a href="#" class="dropdown-toggle" data-toggle="dropdown">Other Languages<b class="caret"></b></a>
+                    <ul class="dropdown-menu">
+                        <li><a href="http://gcc.beta.godbolt.org">C++</a></li>  
```

Comment:
Can you change it to be https or maybe just "//d.beta.godbolt.org/" ? That way we don't lose the https-ness :)
Some other thoughts:
* This only works for the beta site. I bet I forget when I update the main site...
* What about other installations of Compiler Explorer? We probably need to support not having these links for them

I'm wondering if this list needs to be dynamically built from configuration: there's some options in `static/options.json` that comes from `app.js` from the server. Can feasibly use that to hold a map of "other sites" that can then be determined by the server configuration.